src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Machine> {
+    @Override
+    public List<Machine> get() {
```

Comment:
The return type of the `get()` method should be `List<Truck>` instead of `List<Machine>` to match the parameterization of `TruckProducer` with `Truck`.